src/components/card/index.tsx

```diff
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import { memo } from "react";
+import { type CardData } from "@pb/types";
```

Comment:
Could you kindly ensure that there is a space between the various import types, please?